docs/source/tutorials/audio.ipynb

```diff
@@ -435,7 +436,7 @@
     "\n",
     "model = LogisticRegression(C=0.01, max_iter=1000, tol=1e-1, random_state=SEED)\n",
     "\n",
-    "# Generate cross-validated predicted probabilities for each datapoint\n",
+    "num_crossval_folds = 5  # can decrease this value to reduce runtime, or increase it to get better results\n",
     "cv_pred_probs = cross_val_predict(\n",
     "    estimator=model, X=embeddings_array, y=df.label.values, cv=5, method=\"predict_proba\"\n",
     ")"
```

Comment:
`num_crossval_folds` seems to be defined here, but not directly used in `cross_val_predict`

Reply:
good catch!

Reply:
fixed it